spec/genre_spec.rb

```diff
@@ -0,0 +1,26 @@
+require_relative '../musicAlbum/genre'
+
+describe Genre do
+  before :each do
+    @item = Item.new('2022-04-04')
+    @genres = Genre.new 'Nemwel'
+  end
+
+  describe '#new' do
+    it 'Takes in one parameter and returns a Genre object' do
+      @genres.should be_an_instance_of Genre
+    end
+  end
+
+  describe '#name' do
+    it 'Returns the correct name' do
+      @genres.name.should eql 'Nemwel'
+    end
+  end
```

Comment:
[ Optional ] I would suggest you use the `AAA` pattern.
Example: 
**Arrange** -->  @genres = Genre.new 'Nemwel'
**Act**  -->   expected_value = 'Nemwel'
**Assert** -->    expect(@genres.name).to eql(expected_value)

Reply:
Thank you @Achilles-Dev for this suggestion, I will effect it.